### ISSUE TITLE
Texture issue

### ISSUE THREAD

**Issue by riWWE** [Darkosto/SevTech-Ages#1223]
### Before anything else, use the *search* feature!
 * Maybe someone already reported the issue you're experiencing?
 * Maybe you can find the answer to your question by looking at older or closed issues?
 * Have a go at it and see!
 * Please search on the [issue track](../) before creating one.
 
## Issue / Bug

Shading doesnt seem to work. texture might be off as well
## Steps to Reproduce (for bugs)

1.
2.

## Your Environment

* Modpack Version:
* Java Version:
* Launcher Used:
* Resourcepack Enabled?:
* Optifine Installed?:

**Comment by Deamun** [comment]
Having the same problem, thought it was because I had recently tried adding optifine and playing with shaders. Uninstalled and reinstalled, issue is persisting. Enabling a resource pack fixes the issue, however I would prefer not to have a resource pack

**Comment by riWWE** [comment]
For me the problem started after installing 3.0.4 and even after uninstalling and reinstalling just 3.0.3 its still there. weird

**Comment by Deamun** [comment]
I haven't tried donwgrading, might give it a try to check it, if I can't find a decent RP

**Comment by Hebbu** [comment]
I also have this problem :( had to start using texture pack to fix it

**Comment by riWWE** [comment]
Did the problem start after upgrading to 3.0.4?

**Comment by Deamun** [comment]
Yeah didn't realise that as part of my optifine/shader tinkering I updated the pack to 3.0.4, until after I noticed the issue

**Comment by Hebbu** [comment]
the bug seems to only affect vanilla textures

**Comment by Hacksterrenewed** [comment]
Same issue here

**Comment by Peso4ek** [comment]
I can advise you to upgrade to version 3.0.3. I download, check)

**Comment by B00rz** [comment]
Its April 1st, its just Mojang. These look like pre-alpha textures

**Comment by Hacksterrenewed** [comment]
If so - that would be the worst April fools of all.. it completely ruins gameplay for today at the very least

**Comment by Turmfalke2** [comment]
I am not sure it is April the 1st. Or least when I launch the game with my time set to April the 2nd I still encounter this issue.

**Comment by Hacksterrenewed** [comment]
Oh god.. yeah see this.. it's April fools..

https://minecraft.net/en-us/article/java-edition-textures-finally-perfected

**Comment by B00rz** [comment]
i was just posting this, they will revert it back tomorrow probably. Just use a texture pack for now

**Comment by skipabeat123** [comment]
This looks so disqusting! [https://i.imgur.com/U3aegfY.png](url)

**Comment by Peso4ek** [comment]
What kind of Podstava !? The problem is not Passed!

**Comment by justinrusso** [comment]
This is minecraft. What a fun day it is.

**Comment by Jerbearjerry** [comment]
Mine was fine last night until my launcher updated. I was logging in because my Dev said there was a problem. I clicked on SevTech in the Twitch launcher and when I clicked the Play button, the launcher updated. When I got in game and turned off Soartex Modded recourse pack all textures were broken.
Screen shots:


https://puu.sh/zU4OO/f789caedeb.png
https://puu.sh/zU4RP/233de5b4de.png
https://puu.sh/zU4U2/259ef3515f.png
